packages/web-twig/tests/components-fixtures/buttonLinkLoadingDefault.twig

```diff
@@ -0,0 +1 @@
+<ButtonLink href="#" color="primary" isLoading>Loading button link</ButtonLink>
```

Comment:
Check the combination of `isLoading isDisabled`, if the `disabled` attribute remains `false`? And the same for the `web-react` package.